assets/queries/terraform/gcp/stackdriver_logging_disabled/query.rego

```diff
@@ -4,30 +4,31 @@ import data.generic.terraform as tf_lib
 
 CxPolicy[result] {
 	resource := input.document[i].resource.google_container_cluster[primary]
-	not resource.logging_service
+	resource.logging_service == "none"
 
 	result := {
 		"documentId": input.document[i].id,
 		"resourceType": "google_container_cluster",
 		"resourceName": tf_lib.get_resource_name(resource, primary),
 		"searchKey": sprintf("google_container_cluster[%s]", [primary]),
```

Comment:
```suggestion
		"searchKey": sprintf("google_container_cluster[%s].logging_service", [primary]),
```

Reply:
ooh missed that.  thanks.

Reply:
On second thought, rejected, this causes tests to fail, I think in the test case where `logging_service` is undefined?  When I reverted the change, anyways, the tests passed...(also the == "none" test was originally using the searchKey without the .logging_service portion)

Reply:
Again to return the result in this rule `.logging_service` needs to be defined. If not defined there is no result for this rule. This would only be a problem with the use of the `not` operator.